tests/common.sh

```diff
@@ -40,7 +40,7 @@ function build_sketches()
     local build_arg=$3
     local build_dir=build.tmp
     mkdir -p $build_dir
-    local build_cmd="python tools/build.py -b generic -v -k -p $PWD/$build_dir $build_arg "
+    local build_cmd="python tools/build.py -b generic -v -w all -k -p $PWD/$build_dir $build_arg "
```

Comment:
If all warnings are now eliminated, isn't it ok to build with -werror? I'd like to enforce no warnings, sooner rather than later.

Reply:
It's already done below where platform.local.txt is modified.

Reply:
Also, that parameter is not passed to GCC but to build.yp,which passes it to arduino-builder, which then uses it to lookup into a GCC setting array.  Can't send in -werror (it's included now in platform.local.txt that's generated before running the builder.